dcos/http.py

```diff
@@ -86,7 +114,11 @@ def _request(method,
     if 'headers' not in kwargs:
         kwargs['headers'] = {'Accept': 'application/json'}
 
-    verify = _verify_ssl(verify, toml_config)
+    if _is_request_to_dcos(url, toml_config):
+        verify = _verify_ssl(verify, toml_config)
+    else:
+        # Ignore core.ssl_verify if request is outside the DC/OS cluster (DCOS_OSS-618)
+        verify = None
```

Comment:
Instead of doing an if/else here, I would just do:
```
if verify and _is_request_to_dcos(url, toml_config):
    verify = _verify_ssl(verify, toml_config)
```